src/util/table/TablePaginationActions.js

```diff
@@ -13,7 +13,7 @@ import FirstPageIcon from "@material-ui/icons/FirstPage";
 import KeyboardArrowLeft from "@material-ui/icons/KeyboardArrowLeft";
 import KeyboardArrowRight from "@material-ui/icons/KeyboardArrowRight";
 import LastPageIcon from "@material-ui/icons/LastPage";
-import { withStyles } from "@material-ui/core/styles";
+import withStyles from "@material-ui/core/styles/withStyles";
```

Comment:
Just picking a random spot - the changes look fine to me, but I guess I was under the impression that the changes in other PRs were to avoid having to modify our import statements 🤔 
This is mostly a black box for me, so I could definitely be missing something.

Reply:
I'm kinda confused about the same, to be honest. In general, the commonjs version of `import { bar } from foo` seems to pull in all of `foo` and the commonjs version of `import bar from foo/bar` only pulls in `bar`, as expected, but I'm not sure why it's not able to use the ES modules to not use the commonjs version at all. I'll see if doing everything but this PR still has negative effects vs. all of them, perhaps it's an artifact more than anything.

Reply:
Okay, I just tried it again completely without this PR and the sizes are actually slightly smaller. I think I must have somehow tricked myself into thinking this part was necessary when it's actually better handled by the rest. I'll probably close this PR out.

Reply:
Cool, thanks for double checking!